### ISSUE TITLE
distGeo small problem

### ISSUE THREAD

**Issue by helgasoft** [edzer/geosphere#4]
Hope this could be the place to report issues for pkg geosphere ...(?)
I had an error in _distGeo_ and debugged in RStudio.  Found this line of code which did not work:
`line 45:  p[] <- as.numeric(p)`
My parameter p was a character vector.  Line 45 could not convert to numeric, so I got an error right after that.

**Comment by edzer** [comment]
No, this is not the right place; the "active" geosphere source tree is at r-forge.r-project.org. I'm willing to contact the author, but you'll first need to provide a reproducible example, and tell in which file this happened.
